### PR TITLE
ChatConfig: update version tsdoc remarks

### PR DESCRIPTION
**Pull request by nsiskind**
defaults to latest now

TEST=none

### CODE REVIEW

File: src/models/ChatConfig.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ export interface ChatConfig {
    * The version of the chat bot configuration.
    *
    * @remarks
-   * Default to 'STAGING' in Chat API
+   * Default to 'LATEST' in Chat API
```
Comment (yen-tt) [inline src/models/ChatConfig.ts:22]: can you run `npm run build` to re-generate the docs?
Also publish a version for this change

Comment (nsiskind) [reply src/models/ChatConfig.ts:22]: done

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/chat-core",
-  "version": "0.7.4",
+  "version": "0.7.5",
```
Comment (yen-tt) [inline package.json:3]: can you do `npm i` to update package-lock.json as well

Comment (nsiskind) [reply package.json:3]: done

**Review by yen-tt — APPROVED** [review]

**Pull request merged**